src/PrestaShopBundle/Command/SecurityAnnotationLinterCommand.php

```diff
@@ -27,35 +27,155 @@
 namespace PrestaShopBundle\Command;
 
 use PrestaShopBundle\Routing\Linter\Exception\LinterException;
+use PrestaShopBundle\Routing\Linter\SecurityAnnotationLinter;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
 use Symfony\Component\Routing\Route;
 
 /**
  * Checks if all admin routes have @AdminSecurity configured
+ *
+ * @see \PrestaShopBundle\Security\Annotation\AdminSecurity
  */
 final class SecurityAnnotationLinterCommand extends ContainerAwareCommand
 {
+    const ACTION_LIST_ALL = 'list';
+    const ACTION_FIND_MISSING = 'find-missing';
+
+    /**
+     * @param string $expression
+     *
+     * @return string
+     */
+    public static function parseExpression($expression)
+    {
+        $pattern1 = '#\[(.*)\]#';
+        $pattern2 = '#is_granted\((.*),#';
+        $matches1 = [];
+        $matches2 = [];
+        preg_match($pattern1, $expression, $matches1);
+        preg_match($pattern2, $expression, $matches2);
+
+        if (count($matches1) > 1) {
+            return $matches1[1];
+        }
+        if (count($matches2) > 1) {
+            return $matches2[1];
+        }
+
+        return '';
+    }
+
+    /**
+     * @return string[]
+     */
+    public static function getAvailableActions()
+    {
+        return [self::ACTION_LIST_ALL, self::ACTION_FIND_MISSING];
+    }
+
     /**
      * {@inheritdoc}
      */
     public function configure()
     {
+        $description = 'Checks if Back Office route controllers has configured Security annotations.';
+        $actionDescription = sprintf(
+            'Action to perform, must be one of: %s',
+            implode(', ', self::getAvailableActions())
+        );
+
         $this
             ->setName('prestashop:linter:security-annotation')
-            ->setDescription('Checks if Back Office route controllers has configured Security annotations.')
-        ;
+            ->setDescription($description)
+            ->addArgument('action', InputArgument::REQUIRED, $actionDescription);
```

Comment:
Don't use argument, prefer option so you will be able to do:
```
>addOption(
        'action',
        null,
        InputOption::VALUE_REQUIRED,
        'Action to perform',
        [self::ACTION_LIST_ALL, self::ACTION_FIND_MISSING]
    )

Reply:
But I expect an argument, not an option 😛 
We run `php vendor/bin/php-cs-fixer fix` and not `php vendor/bin/php-cs-fixer --action=fix`
Same here

Reply:
You're right 👍